src/VisualStudio/Core/Test/Progression/SearchGraphQueryTests_NavigateTo.vb

```diff
@@ -81,7 +81,7 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.Progression
             End Using
         End Function
 
-        <WpfFact>
+        <WpfFact(Skip:="https://github.com/dotnet/roslyn/issues/71290")>
```

Comment:
I created the issue to track this. 
From the CI history, it has been broken for a long time. I believe it is not related to this PR. Skip this to unblock the LOC

Reply:
This was removed from this PR.